open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconManagerFrame.java

```diff
@@ -123,10 +146,29 @@ public IconManagerFrame(Toolbox tb)
         createTreePopupMenuItems(treePopupMenu);
         myIconRegistry.addListener(this);
     }
-
-    /**
-     * Shows the icon builder dialog.
-     */
+	
+	/**
+	 * Initiates icon resizing.
+	 * 
+	 * @param choice
+	 *            the {@link #JButton reduceButton/increaseButton}
+	 */
+	private void iconSizeChanger(boolean choice) 

```

Comment:
Can you rename the argument to something that indicates what the choice is?  Maybe 'enlarge' or something like that?  Also rename the method to chageIconSize, setIconSize, etc